src/Microsoft.OData.Core/Batch/ODataBatchWriter.cs

```diff
@@ -590,7 +592,9 @@ protected void SetState(BatchWriterState newState)
         /// <returns>A task that represents the asynchronous write operation.</returns>
         protected virtual Task WriteStartBatchImplementationAsync()
         {
-            return TaskUtils.GetTaskForSynchronousOperation(this.WriteStartBatchImplementation);
+            return TaskUtils.GetTaskForSynchronousOperation(
+                (thisParam) => thisParam.WriteStartBatchImplementation(),

```

Comment:
what's the difference in doing this from what was there before? 
`this.WriteStartBatchImplementation`